core/src/test/java/org/elasticsearch/search/aggregations/metrics/HDRPercentileRanksIT.java

```
@@ -56,13 +58,66 @@
 import static org.hamcrest.Matchers.notNullValue;
 import static org.hamcrest.Matchers.sameInstance;
 
-/**
- *
- */
-public class HDRPercentileRanksTests extends AbstractNumericTestCase {
+public class HDRPercentileRanksIT extends AbstractNumericTestCase {
+
     @Override
     protected Collection<Class<? extends Plugin>> nodePlugins() {
-        return Collections.singleton(GroovyPlugin.class);
+        return Collections.singleton(CustomScriptPlugin.class);
+    }
+
+    public static class CustomScriptPlugin extends MockScriptPlugin {
+
+        @Override
+        protected Map<String, Function<Map<String, Object>, Object>> pluginScripts() {
+            Map<String, Function<Map<String, Object>, Object>> scripts = new HashMap<>();
+
+            scripts.put("20 - _value", vars -> {
+                double value = (double) vars.getOrDefault("_value", 0.0d);
```

It feels weird to default here. Maybe just let the NPE bubble up?